wallet-new/src/Cardano/Wallet/API/V1/Swagger.hs

```diff
@@ -1083,7 +1096,8 @@ api (compileInfo, curSoftwareVersion) walletAPI mkDescription = toSwagger wallet
     , deSoftwareVersion       = fromString $ show curSoftwareVersion
     }
   & info.license ?~ ("MIT" & url ?~ URL "https://raw.githubusercontent.com/input-output-hk/cardano-sl/develop/lib/LICENSE")
-  & paths %~ (POST, "/api/internal/apply-update") `setDescription` applyUpdateDescription
-  & paths %~ (POST, "/api/internal/postpone-update") `setDescription` postponeUpdateDescription
+  & paths %~ (POST,   "/api/internal/apply-update")       `setDescription` applyUpdateDescription
+  & paths %~ (POST,   "/api/internal/postpone-update")    `setDescription` postponeUpdateDescription
   & paths %~ (DELETE, "/api/internal/reset-wallet-state") `setDescription` resetWalletStateDescription
-  & paths %~ (POST, "/api/v1/transactions/fees") `setDescription` estimateFeesDescription
+  & paths %~ (POST,   "/api/v1/transactions/fees")        `setDescription` estimateFeesDescription
+  & paths %~ (GET,    "/api/v1/addresses")                `setDescription` getAddressDescription
```

Comment:
Uhm, thorny -- I guess it's a question more for @KtorZ : how resilient is our magic combinator when it comes to "ambiguous" endpoints? Here `/api/v1/addresses` might refer both to the endpoint "give me all the addresses" but also to `/api/v1/addresses/{addressId}` which means "give me just this single address".

I have the suspect the combinator will try to apply this description to both endpoint, which is obviously not correct. Am I right? 🤔

Reply:
> Here /api/v1/addresses might refer both to the endpoint "give me all the addresses" but also to /api/v1/addresses/{addressId} which means "give me just this single address"

Na. `GET /api/v1/addresses` unambiguously refers to the list addresses endpoint. The single get would be referred to as: `GET  /api/v1/addresses/{addressId}` ! So no issue here, that's resilient :muscle: 

(as shown in the doc example: https://github.com/input-output-hk/cardano-sl/blob/develop/wallet-new/src/Cardano/Wallet/API/V1/Swagger.hs#L157-L161 :trollface: )